src/lib/components/ui/dialog/dialog-content.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+	import { Dialog as DialogPrimitive } from 'bits-ui';
+	import X from 'lucide-svelte/icons/x';
+	import * as Dialog from './index.js';
+	import { cn, flyAndScale } from '$lib/utils.js';
+
+	interface $$Props extends DialogPrimitive.ContentProps {
+		includeX: boolean;
+	}
+
+	let className: $$Props['class'] = undefined;
+	export let transition: $$Props['transition'] = flyAndScale;
+	export let transitionConfig: $$Props['transitionConfig'] = {
+		duration: 200
+	};
+	export { className as class };
+	export let includeX: $$Props['includeX'] = true;
+</script>
+
+<Dialog.Portal>
+	<Dialog.Overlay />
+	<DialogPrimitive.Content
+		{transition}
+		{transitionConfig}
+		class={cn(
+			'bg-background fixed left-[50%] top-[15%] z-50 grid w-full max-w-lg translate-x-[-50%] gap-4 border p-6 shadow-lg sm:rounded-lg md:w-full',
```

Comment:
I failed to stick to the `shadcn-svelte` colours like `background, muted, accent...`, and as a result, class names like `bg-background` produce no output.

I tried to help out in the https://github.com/shyakadavis/geist/tree/dev branch and added the equivalent classes if you want to copy-paste.

Reply:
I noticed that I'll need to purge dead classes. I think it may be good to add a part for this in a contribution guide of sorts.

Reply:
Ah, nice suggestion. I'll definitely a contribution guide tonight. Thanks.